build/Packages.targets

```diff
@@ -13,6 +13,7 @@
       https://dotnet.myget.org/F/project-system-interop/api/v3/index.json;
       https://vside.myget.org/F/vs-impl/api/v3/index.json;
       https://vside.myget.org/F/vssdk/api/v3/index.json;
+      C:\Users\tomescht\Desktop\LocalPackageSource
```

Comment:
Intentional?

Reply:
Intentional but temporary. The current change depends on CPS packages that only exist on my local system. :-)

Once the CPS side is merged in to the proper VS branch I can update the change to use the real package version, and then I will remove this.